ng-dev/release/publish/actions/shared/cut-prerelease.ts

```diff
@@ -0,0 +1,95 @@
+/**
```

Comment:
FYI: This file and the followed ones are 99% the same, except:

* For the abstract class members
* Removal of `isActive` — that's left to the implementing classes
* Usage of the abstract members when determining `newVersion` and `releaseNotesCompareVersion`.